fastos/src/vespa/fastos/unix_dynamiclibrary.cpp

```diff
@@ -56,7 +56,7 @@ FastOS_UNIX_DynamicLibrary::Close()
     bool retcode = true;
 
     if (IsOpen()) {
-        retcode = (dlclose(_handle) == TRUE);
+        retcode = (dlclose(_handle) == 0);
```

Comment:
I also shook my eyes on this one. Either one one checks the return code or TRUE is 0.....